CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 * [CHANGE] Change default value for `-distributor.ha-tracker.max-clusters` to `100` to provide a DoS protection. #2465
 * [CHANGE] Experimental block upload API exposed by compactor has changed: Previous `/api/v1/upload/block/{block}` endpoint for starting block upload is now `/api/v1/upload/block/{block}/start`, and previous endpoint `/api/v1/upload/block/{block}?uploadComplete=true` for finishing block upload is now `/api/v1/upload/block/{block}/finish`. #2486
 * [CHANGE] Compactor: changed `-compactor.max-compaction-time` default from `0s` (disabled) to `1h`. When compacting blocks for a tenant, the compactor will move to compact blocks of another tenant or re-plan blocks to compact at least every 1h. #2514
+* [CHANGE] Distributor: removed previously deprecated `extend_writes` (see #3719) YAML key from the Distributor config. #2551
```

Comment:
Doesn't seem like that issue/PR exists in Mimir. There is a related Cortex issue but it doesn't seem like it deprecates the option.

Reply:
Just force-pushed, I referenced the wrong issue. Good catch :)